data/datasets/__init__.py

```diff
@@ -13,3 +13,8 @@
     "youtube_subs_howto100M": "totuta/youtube_subs_howto100M",
     "iapp_wiki_qa_squad": "wannaphong/iapp_wiki_qa_squad_oa",
 }
+
+SAFETY_DATASETS = {
+    "prosocial-dialog": "allenai/prosocial-dialog",
+    "prosocial-confessions": "shahules786/prosocial-confessions",
```

Comment:
How these two datasets can be fit in one training process if `RESPONSE` column isn't provided in `prosocial-confessions` dataset?

Reply:
The current response for the user's prompt is not taken into consideration while modeling. Response is used to form contextual information about past conversations. In prosocial confessions, each example is a single conversation indicated by `episode_done` set to True.